src/glb-healthcheck/TcpHealthChecker.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"expvar"
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"net"
+	"time"
+	 )
+
+var (
+	tcpCounters = expvar.NewMap("TcpHealthChecker")
+)
+
+type TcpHealthChecker struct {
+	 checkTimeout time.Duration
+}
+func OpenConnection(resultChannel HealthResultStream, ip_port string) HealthResultStream {
+ 	 ch := make(HealthResultStream, 1)
+
+	 go func() {
+	 	 c, err := net.Dial("tcp", ip_port)
+	 	 if err != nil {
+		 	fmt.Println(err)
```

Comment:
Would be good to remove this or move it to use the structured logging library (though I think it would get logged below already).

Reply:
Agreed.